File: demo/src/main/java/br/com/stickyindexsample/view/FavoritesView.java
package br.com.stickyindexsample.view;


import android.app.Activity;
import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import br.com.stickyindexsample.R;

/**
 * Created by edgar on 6/7/15.
 */
public class FavoritesView extends Fragment {

    private Activity mActivity;
    private RecyclerView recyclerView;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_favorites, container, false);
    }

    @Override
    public void onAttach(Activity activity) {
        super.onAttach(activity);
        mActivity = activity;
    }
}

File: demo/src/main/java/br/com/stickyindexsample/view/FavoritesView.kt
package br.com.stickyindexsample.view


import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import br.com.stickyindexsample.R

/**
 * Created by edgar on 6/7/15.
 */
class FavoritesView : Fragment() {
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_favorites, container, false)
    }
}
